lib/grinder_tools.dart

```diff
@@ -16,10 +15,14 @@ import 'package:cli_util/cli_util.dart' as cli_util;
 import 'package:which/which.dart';
 
 import 'grinder.dart';
+import 'src/run.dart' as r;
```

Comment:
Changed to `run_lib`.  I originally had `run`, but it conflicted with `Pub.run` etc.